RELEASE_NOTES.md

```diff
@@ -1,3 +1,47 @@
+## Cookstyle 5.7
+
+### 5 New Chef Cops
+
+#### ChefDeprecations/ResourceOverridesProvidesMethod
+
+The `ChefDeprecations/ResourceOverridesProvidesMethod` cop detects overriding the `provides?` method in a resource's provider without also calling `provides :SOME_PROVIDER_NAME`. This will cause failures in Chef Infra Client 13 and later.
```

Comment:
`provides?` should just get dropped though in favor of the provides dsl line.

Reply:
Updated